app/queries/follow_ups_query.rb

```diff
@@ -0,0 +1,54 @@
+class FollowUpsQuery
+  def initialize(region, period_type, group_by: nil)
+    @region = region
+    @period_type = period_type
+    @group_by = group_by
+    @formatter = lambda { |v| @period_type == :quarter ? Period.quarter(v) : Period.month(v) }
+  end
+
+  def hypertension
```

Comment:
This is partially replicated in the `with` method, can we just call `with` instead?

Reply:
Can we do one of:
- Keep all the `FollowUpsQuery` scopes in Patient and remove `.hypertension` here
- Move all these ^ scopes to `FollowUpsQuery`

Doing both is a bit confusing.